services/datamanager/builtin/file_deletion.go

```diff
@@ -0,0 +1,111 @@
+package builtin
+
+import (
+	"context"
+	"errors"
+	"io/fs"
+	"math"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"go.viam.com/rdk/logging"
+
+	"github.com/ricochet2200/go-disk-usage/du"
+	"go.viam.com/rdk/services/datamanager/datacapture"
+	"go.viam.com/rdk/services/datamanager/datasync"
+)
+
+// TODO change these values back to what they should be
+const (
+	fileDeletionThreshold    = .2
+	captureDirRatioThreshold = .1
```

Comment:
`captureDirToFSUsageRatio`? Thinking through what's descriptive here, since initially took me a bit to understand the relationship between the example 0.2 and 0.4 in your PR description.

Reply:
That is a much better name lol